changelog.md

```diff
@@ -109,6 +109,16 @@
 
 - Using an unnamed break in a block is deprecated. This warning will become an error in future versions! Use a named block with a named break instead.
 
+- Several Standard libraries are moved to nimble packages, use `nimble` to install them:
+  - `std/punycode` => `pkg/punycode`
+  - `std/asyncftpclient` => `pkg/asyncftpclient`
+  - `std/smtp` => `pkg/smtp`
+  - `std/db_common` => `db_connector/db_common`
+  - `std/db_sqlite` => `db_connector/db_sqlite`
+  - `std/db_mysql` => `db_connector/db_mysql`
+  - `std/db_postgres` => `db_connector/db_postgres`
+  - `std/db_odbc` => `db_connector/db_odbc`
```

Comment:
I think this list is a bit unclear as it doesn't directly say what to _install_ with Nimble - `pkg/punycode` or just `punycode`?

Reply:
Thanks for your suggestions.